jamal-api/pom.xml

```diff
@@ -1,49 +1,38 @@
 
 <project xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xmlns="http://maven.apache.org/POM/4.0.0"
```

Comment:
Please do not reformat `pom.xml` which is generated from the `pom.xml.jam`. I cannot exclude individual files from the pull. (I can, but it is some extra work.)